python/pyspark/sql/tests/test_arrow.py

```diff
@@ -315,6 +346,17 @@ def check_create_data_frame_to_pandas_timestamp_ntz(self, arrow_enabled):
                 pdf = df.toPandas()
             assert_frame_equal(origin, pdf)
 
```

Comment:
Each Arrow test here is based on a pandas test above it.

- `test_create_data_frame_to_arrow_timestamp_ntz` is based on `test_create_data_frame_to_pandas_timestamp_ntz`
- `test_createDataFrame_arrow_respect_session_timezone` is based on `test_createDataFrame_pandas_respect_session_timezone`

... and so on. I renamed the pandas ones with `pandas` in the name for clarity.